client/App/Containers/LoggingSamples/LoggingSamplesScreen.js

```diff
@@ -33,8 +48,9 @@ class LoggingSamplesScreen extends React.Component {
             <View style={Style.logoContainer}>
               <Image source={Images.Location} resizeMode={'contain'} />
             </View>
-            <View style={Helpers.center}>
+            <View >
               <Text style={[Style.text]}>Thanks for helping us mapping the Hospital!</Text>
+              <Text style={[Style.counter]}>Scans: {this.state.sampleCounter}</Text>
```

Comment:
how about use destruct like we did with props on top of render?

Reply:
Don't understend

Reply:
explain plsssss

Reply:
on top of render:
``` const { sampleCounter } = this.state; ```

then you can: 
```suggestion
              <Text style={[Style.counter]}>Scans: {sampleCounter}</Text>
```
which will be more readable

Reply:
וואו איזה פאוור יוזר של גיטהאב אתה